### DISCUSSION TITLE
Main UI improvements

### DISCUSSION THREAD

**Discussion by newhinton** [PaulWoitaschek/Voice#1586] [Ideas]
Currently i stumble over many issues with the new UI, some of them have been helped with the new search.

I would like to propose some ideas that woild help with that.

To my problems:

I have mainly audiobooks, roughly 100. A lot of them are series.
This leads to a basically unusable main view. All books are shown, finding the new or done category is hard to impossible. Also the new category is sorted alphabetically, which makes it unintuitive to find newly added books.

I suggest re-adding the drop-down as to limit the amount of books shown per category. However, i am not sure if those categories are the best way to display books, so in addition to the current "categories" i propose two additional groupings: author and series

Both can act as the main view, by preference of the user. Author creates a category for each author, only showing 2-4 books at max, sorted by alphabet.
The same for series, except they should not be sorted by alphabet but by their respective place.

This concept should also work for podcasts. This would make navigating voice much easier than it currently is, with very little drawback.

**Comment by mesonium** [comment]
Related: https://github.com/PaulWoitaschek/Voice/discussions/1214, https://github.com/PaulWoitaschek/Voice/discussions/1543, https://github.com/PaulWoitaschek/Voice/discussions/1314
